### PR TITLE
[WIP] 6.x

### PR DESCRIPTION
**Pull request by oleg-andreyev**
- Master is 6.x for development
- Merge stuff from 5.x to ensure that master is up-to-date
- Using this PR for attempt of upgrade to Symfony 7.x

### CODE REVIEW

**Comment by Bukashk0zzz** [comment]
Any reason why this not merged? 
@DamienHarper

**Review by DamienHarper — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,75 +1,83 @@
...
+    {
+      "name": "Damien Harper",
+      "email": "damien.harper@gmail.com"
     }
+  ],
+  "require": {
+    "php": ">=8.1",
+    "ext-json": "*",
+    "damienharper/auditor": "dev-master",
+    "doctrine/annotations": "^1.8|^2.0",
```
Comment (maxhelias) [inline composer.json:23]: “doctrine/annotations” has been restored while its support has been removed here : https://github.com/DamienHarper/auditor-bundle/pull/337

Comment (maxhelias) [reply composer.json:23]: See : https://github.com/DamienHarper/auditor-bundle/pull/419

Comment (oleg-andreyev) [reply composer.json:23]: it could be a bad merge, or it was required for something at that moment.

**Pull request merged**